library/ssl_tls12_client.c

```diff
@@ -1475,8 +1473,7 @@ static int ssl_parse_server_hello( mbedtls_ssl_context *ssl )
     }
 #endif
 
-    if( comp != MBEDTLS_SSL_COMPRESS_NULL
-      )
+    if( comp != MBEDTLS_SSL_COMPRESS_NULL )
```

Comment:
The equivalent line for TLS 1.3 hard-codes 0 instead of using the constant MBEDTLS_SSL_COMPRESS_NULL - would be nice to fix this to use the constant while you're here.

Reply:
Same again in `ssl_tls13_write_server_hello_body`